rubric.md

```diff
@@ -0,0 +1,30 @@
+# Scoring Rubric
+This document outlines the criteria that will be used to judge the entries for the hackathon.
+
+1. Relevance to Societal Challenges (1-20 points)
+   * How well does the project address a societal challenge or improve people's resiliency?
+   * How well aligned is the project with one or more UN Sustainability Goals?
+2. Innovative Use of AI (1-20 points)
+   * How creatively has the team applied AI to solve a problem or enhance a system?
+   * Is the AI model efficient in generating the desired outcomes? 
+3. Technical Execution (1-20 points)
+   * How well has the team implemented the AI model?
+   * Is the project easy to maintain and operate by the non-profit?
+4. Real-world Impact (1-20 points)
+   * How significant is the potential impact of the project on the targeted societal issue or community?
+   * What is the likelihood of the solution making a tangible difference?
+5. Scalability and Sustainability (1-20 points)
+   * How well can the project scale to meet larger demands or serve larger populations?
+   * How sustainable is the solution in terms of resources required for maintenance and operation?
+6. Usability and Accessibility (1-20 points)
+   * How user-friendly is the project? Is it accessible to the target audience, including people with diverse abilities?
+   * If applicable, is the interface intuitive and easy to navigate?
+7. Presentation and Documentation (1-20 points)
+   * How effectively has the team communicated their idea, implementation strategy, and potential impact?
+   * Is the project well-documented, allowing for understanding and potential replication of the work?
+ 
```

Comment:
Remove this extra line break? LGTM otherwise

Reply:
Resolved in 79293eb4a919c4988f57d3c78aa868bec9ded499